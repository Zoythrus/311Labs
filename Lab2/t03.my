t annnce shld cld rnd thght witht 

annnce cld rnd shld t thght witht 
