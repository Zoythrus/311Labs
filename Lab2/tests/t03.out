annnce cld rnd shld t thght witht y 
